src/target/riscv/riscv.h

```diff
@@ -297,6 +297,10 @@ struct riscv_info {
 	yes_no_maybe_t vsew64_supported;
 
 	bool range_trigger_fallback_encountered;
+
+	bool riscv_ebreakm;
+	bool riscv_ebreaks;
+	bool riscv_ebreaku;
```

Comment:
I like this change. Can you mention it in the description of the PR?

Reply:
Commit message updated. Addressed.